src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -997,7 +997,11 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             // _cursorTimer doesn't exist, and it would never turn on the
             // cursor. To mitigate, we'll initialize the cursor's 'on' state
             // with `_focused` here.
-            _core.CursorOn(_focused);
+            _core.CursorOn(_focused || DisplayCursorWhileBlurred);
+            if (DisplayCursorWhileBlurred)
+            {
+                _cursorTimer->Start();
+            }

```

Comment:
This doesn't work if the `blinkTime` is `INFINITE`?

Reply:
nope, but that's not new from this code.